config/install/responsive_image.styles.full_responsive.yml

```diff
@@ -0,0 +1,37 @@
+uuid: 77b69a0c-cd0a-4916-8844-d0acc355dded
+langcode: en
+status: true
+dependencies:
+  config:
+    - image.style.responsive_large
+    - image.style.responsive_medium
+    - image.style.responsive_small
+  module:
+    - stanford_media
+_core:
+  default_config_hash: 87YDNYUvaTv6bZRJVF4DsV7apqKNcQIY5xTHMC97ZjM
+id: full_responsive
+label: 'Full Responsive'
+image_style_mappings:
+  -
+    breakpoint_id: stanford_media.xl
```

Comment:
These will likely need to be re-mapped to match the breakpoints in jumpstart_ui/stanford_basic

Reply:
https://stanfordits.atlassian.net/browse/D8CORE-334